package-support-policy.md

```diff
@@ -0,0 +1,372 @@
+| System.Net.Http.Json                                         | 8.0                  |
+| System.Net.Http.WinHttpHandler                               | 8.0                  |
+| System.Numerics.Tensors                                      | 8.0                  |
+| System.Reflection.Context                                    | 8.0                  |
+| System.Reflection.Metadata                                   | 8.0                  |
+| System.Reflection.MetadataLoadContext                        | 8.0                  |
+| System.Resources.Extensions                                  | 8.0                  |
+| System.Runtime.Caching                                       | 8.0                  |
+| System.Runtime.Serialization.Schema                          | 8.0                  |
+| System.Security.Cryptography.Cose                            | 8.0                  |
+| System.Security.Cryptography.Pkcs                            | 8.0                  |
+| System.Security.Cryptography.ProtectedData                   | 8.0                  |
+| System.Security.Cryptography.Xml                             | 8.0                  |
+| System.Security.Permissions                                  | 8.0                  |
+| System.ServiceModel.Syndication                              | 8.0                  |
+| System.ServiceProcess.ServiceController                      | 8.0                  |
+| System.Speech                                                | 8.0                  |
+| System.Text.Encoding.CodePages                               | 8.0                  |
+| System.Text.Encodings.Web                                    | 8.0                  |
+| System.Text.Json                                             | 8.0                  |
+| System.Threading.AccessControl                               | 8.0                  |
+| System.Threading.Channels                                    | 8.0                  |
+| System.Threading.RateLimiting                                | 8.0                  |
+| System.Threading.Tasks.Dataflow                              | 8.0                  |
+| System.Windows.Extensions                                    | 8.0                  |
+| System.Security.AccessControl                                | 6.0                  |
+| Microsoft.Diagnostics.Tracing.EventSource.Redist             | 6.0                  |
+| Microsoft.IO.Redist                                          | 6.0                  |
+
+### Others
+
+These packages are fully unsupported:
+
+| Package                             | Last shipped release |
+|-------------------------------------|----------------------|
+| System.Diagnostics.Tracing          | 1.1                  |
+| System.Reflection.Emit              | 3.1                  |
+| System.Reflection.Emit.ILGeneration | 3.1                  |
+| System.Reflection.Emit.Lightweight  | 3.1                  |
```

Comment:
Looking at these again:

- For System.Diagnostics.Tracing we've been providing workarounds and don't want to support this package anymore. So I could put it in a separate table indicating this info.

For System.Reflection.Emit* it seems we made them in-box (there used to be a `pkg/` folder in 3.1, and we didn't add the `GeneratePackageOnBuild` in 5.0). Which means I could move them to the `## In-box` section above instead.

Reply:
Based on the other feedback, this table will change considerably so I'll resolve this.